### PR TITLE
UI Design updating

### PR DESCRIPTION
**Pull request by Dipeshdahiya**
#3 I have updated the design , basically the navbar , the search engine , the collection columns , and the footer

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -12,19 +12,25 @@ body{
...
-  display: inline-flex;
+  position: sticky;
+  top: 0%;
+  display: flex;
   justify-content: space-between;
   padding: 20px;
-  padding-right:15% ;
-  background-color: black;
+  align-items: center;
+  height: 10vh;
```
Comment (harshit1142) [inline styles.css]: Increase the height of navbar

**Comment by Dipeshdahiya** [comment]
I have adjusted the navbar size , also please add the hacktoberfest lables and tell if any other issue needed

**Review by harshit1142 — APPROVED** [review]

**Comment by harshit1142** [comment]
@all-contributors please add @Dipeshdahiya  for design

**Comment by Dipeshdahiya** [comment]
@harshit1142 Is their is anything else for me to do ?

**Comment by harshit1142** [comment]
@Dipeshdahiya No.

**Pull request merged**